draft-js-mention-plugin/src/MentionSuggestions/Entry/defaultEntryComponent.js

```diff
@@ -5,6 +5,7 @@ const defaultEntryComponent = (props) => {
   const {
     mention,
     theme,
+    isFocused, // eslint-disable-line no-unused-vars
```

Comment:
so, you're not actually removing any props, how is this fixing this issue?

Reply:
The issue is that the `parentProps` which are being passed through to the `<div>` [(Line 14)](https://github.com/hosmelq/draft-js-plugins/blob/cfdbb0e1f7cb924f061e9b070dfe047c121f7e43/draft-js-mention-plugin/src/MentionSuggestions/Entry/defaultEntryComponent.js#L14) include the `isFocused` property, which isn't a valid prop DOM element prop, so React warns about it.

![image](https://user-images.githubusercontent.com/6840861/35508100-0a2a4682-04e7-11e8-96ad-c534cbfd5542.png)

By explicitly declaring `isFocused` when destructuring the props @hosmelq is ensuring that `isFocused` is not included in `parentProps`, which contains all props which haven't been explicitly unpacked.

Reply:
The same thing could be achieved by adding `delete parentProps.isFocused` on line 12. Both methods achieve the same end result and fix the issue.

Reply:
It seems `isFocused` was added at #878 
the guy was just disabled linter, any problem to merge this PR @juliankrispel ?

Reply:
It seems `isFocused` added from #878 
Any problem with disabled linter @juliankrispel ? Is it ok to merge?

Reply:
Ah I see. Look forward to seeing this merged.